CMakePresets.json

```diff
@@ -19,23 +19,23 @@
       }
     },
     {
-      "name": "ci-linux",
+      "name": "ci-ubuntu-latest",
```

Comment:
A bit unfortunate that this uses the GH image names. For `package-` it is tied to the conda platform names. Would it make sense to use those here as well?

Reply:
We could do that, but then we need a translation table in the github workflow. Is it worth the hassle? You are worried about a move away from github?

Reply:
> You are worried about a move away from github?

Not really. Adjusting preset names would be a trivial change. It's more about having consistent names in the presets to make it easier to get an overview of how they relate to each other.
But I don't mind, this is just a minor comment.

Reply:
Then keeping names same as on the workflow may actually be simpler to understand?

Reply:
Could be